pom.xml

```diff
@@ -404,8 +405,7 @@
             <plugin>
                 <groupId>org.apache.maven.skins</groupId>
                 <artifactId>maven-fluido-skin</artifactId>
-                <!-- Must ALSO update this version in src/site/site.xml file as that file doesn't inherit from this pom! -->
-                <version>1.11.1</version>
+                <version>${fluido.version}</version>
```

Comment:
This entire block looks totally wrong. The skin is not a plugin. What is the purpose here?